src/components/Authentification/GoogleSignIn.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { GoogleSignInType } from '../ToDoList/types';
+
+function GoogleSignIn(props: GoogleSignInType): JSX.Element {
```

Comment:
If you don't have a state, rendering logic or other specific functionality, you don't need such a component. You can try to create a universal stylized button and pass the business logic through props.